tests/test_move_embargo_args.py

```diff
@@ -4,6 +4,9 @@
 import os
 import tempfile
 
+import lsst.utils as utils
+from typing import Union
```

Comment:
This is now replaced by `|`.

Reply:
You can run `ruff --select=UP --fix .` to automatically do this fix (and a few others). You can constrain it to just the typing fixes using the relevant UPxx code.